pkgs/top-level/all-packages.nix

```diff
@@ -23445,6 +23445,8 @@ with pkgs;
 
   go-font = callPackage ../data/fonts/go-font { };
 
+  graphite-gtk-theme = callPackage ../data/themes/graphite { };
```

Comment:
I remember that years ago there was a discussion about naming themes. The conclusion was that it would be better to use only the theme name (instead of the package name) as the directory name.

_If I find the link to the discussion, I will place it here._

Reply:
The discussion is here: https://github.com/NixOS/nixpkgs/pull/16328

Reply:
This is ironic because right below your package `gruvbox-dark-gtk` is using its full name :sweat_smile: .

But sure enough, seems fair.

Reply:
Since then I have followed this orientation, although I would personally prefer to name the directory following the package name.

Reply:
BTW, the reason I asked this is because I find it difficult to find the derivations for files that don't match their package. I tried to fuzzy search for `graphite-gtk-theme` and of course I didn't find any file named as this.

Reply:
I think this orientation is not documented anywhere besides the mentioned PR. Maybe it does not have to be followed. Or we can re-discuss it.

Reply:
I think it is fine to merge this as-in.

Probably if we want to fix it we first need to open an issue and have some discussion. Afterwards maybe a PR fixing it treewide.